src/Esfa.Vacancy.Register.Domain/Validation/ErrorMessages.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Esfa.Vacancy.Register.Domain.Validation
+{
+    public static class ErrorMessages
+    {
+        public static class SearchApprenticeships
+        {
+            public const string SearchApprenticeshipParametersIsNull  = "At least one search parameter is required.";
+
+            public const string StandardAndFrameworkCodeNotProvided   = "At least one of StandardCodes or FrameworkCodes is required.";
+            public const string StandardCodeNotInt32                  = "{0} is invalid, expected a number.";
+            public const string FrameworkCodeNotInt32                 = "{0} is invalid, expected a number.";
+            //public const string PageSizeOutsideRange                = "this message is auto-generated by fluent validator";
```

Comment:
We have agreed on default messages produced by Fluent validation in straight forward cases.